src/styles/modules/nav.scss

```diff
@@ -3,13 +3,18 @@
    justify-content: center; 
    box-sizing: border-box;
    background-color: #313030;
+   & a:hover {
```

Comment:
why are you using styles for `a` tag, in spite of having class for your `a`, called `.nav-item` ?